### ISSUE TITLE
got sigterm without errors when trying to start homebridge

### ISSUE THREAD

**Issue by the-eternel** [oznu/homebridge-config-ui-x#991]
**Describe The Bug:**

homebridge doesn't start with homebridge-config-ui-x plugin (no bug reported by log, just a sigterm and I don't know why)

**To Reproduce:**
install homebridge & homebridge-config-ui-x on ubuntu and start after set config.json

**Expected behavior:**

**Logs:**

```
[06/12/2020 à 21:24:59] Loaded config.json with 0 accessories and 1 platforms.
[06/12/2020 à 21:24:59] ---
[06/12/2020 à 21:24:59] Loaded plugin: homebridge-config-ui-x@4.35.0
[06/12/2020 à 21:24:59] Registering platform 'homebridge-config-ui-x.config'
[06/12/2020 à 21:24:59] ---
[06/12/2020 à 21:24:59] Loading 1 platforms...
[06/12/2020 à 21:24:59] [Config] Initializing config platform...
[06/12/2020 à 21:24:59] [Config] Spawning homebridge-config-ui-x with PID 13461
Setup Payload:
X-HM://0023ISYWY4A63
Scan this code with your HomeKit app on your iOS device to pair with Homebridge:
                                        
Or enter this code with your HomeKit app on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 031-45-154 │     
    └────────────┘     
                       
  HAP-NodeJS:EventedHTTPServer Server listening on port 39733 +0ms
[06/12/2020 à 21:24:59] Homebridge is running on port 39733.
  avvio exec bound root +0ms
  avvio enqueue bound root bound _after +2ms
  avvio enqueue bound root /usr/lib/node_modules/homebridge-config-ui-x/node_modules/fastify-multipart/index.js +0ms
  follow-redirects options {
  protocol: 'https:',
  maxRedirects: 21,
  maxBodyLength: 10485760,
  path: '/homebridge/verified/master/verified-plugins.json',
  method: 'GET',
  headers: {
    Accept: 'application/json, text/plain, */*',
    'User-Agent': 'homebridge-config-ui-x'
  },
  agent: null,
  agents: { http: undefined, https: null },
  auth: undefined,
  hostname: 'raw.githubusercontent.com',
  port: null,
  nativeProtocols: {
    'http:': {
      _connectionListener: [Function: connectionListener],
      METHODS: [Array],
      STATUS_CODES: [Object],
      Agent: [Function],
      ClientRequest: [Function: ClientRequest],
      IncomingMessage: [Function: IncomingMessage],
      OutgoingMessage: [Function: OutgoingMessage],
      Server: [Function: Server],
      ServerResponse: [Function: ServerResponse],
      createServer: [Function: createServer],
      validateHeaderName: [Function: hidden],
      validateHeaderValue: [Function: hidden],
      get: [Function: get],
      request: [Function: request],
      maxHeaderSize: [Getter],
      globalAgent: [Getter/Setter]
    },
    'https:': {
      Agent: [Function: Agent],
      globalAgent: [Agent],
      Server: [Function: Server],
      createServer: [Function: createServer],
      get: [Function: get],
      request: [Function: request]
    }
  }
} +0ms
[06/12/2020 à 21:25:00] Got SIGTERM, shutting down Homebridge...

```

**Homebridge Config:**

```json
{
    "platforms": [
        {
          "platform": "config",
          "name": "Config",
          "port": 8185,
          "sudo": false
        }
    ]
}
```

**Screenshots:**

**Environment:**

* **Node.js Version**: v14.15.1
* **NPM Version**: 6.14.8
* **Homebridge Version**: 1.1.6
* **Homebridge Config UI X Version**:  4.35
* **Operating System**: Ubuntu
* **Process Supervisor**:  none

**Comment by oznu** [comment]
Uninstall / re-install:

```
# cleanup
sudo npm uninstall -g homebridge-config-ui-x

# reinstall
sudo npm install -g --unsafe-perm homebridge-config-ui-x
```

**Comment by the-eternel** [comment]
> Uninstall / re-install:
> 
> ```shell
> # cleanup
> sudo npm uninstall -g homebridge-config-ui-x
> sudo npm cache clean --force
> 
> # reinstall
> sudo npm install -g --unsafe-perm homebridge-config-ui-x
> ```

I already did that sevral times and it didn't change anything

**Comment by sticksen** [comment]
I´m having the exact same problem.
Environment:

Node.js Version: v15.4.0
NPM Version: 7.0.15
Homebridge Version: 1.1.6
Homebridge Config UI X Version: 4.35
Operating System: Mac OS X 10.15.7
Process Supervisor: none

**Comment by oznu** [comment]
> Node.js Version: v15.4.0
> NPM Version: 7.0.15

This version is not supported. Please install the current LTS.

**Comment by sticksen** [comment]
> > Node.js Version: v15.4.0
> > NPM Version: 7.0.15
> 
> This version is not supported. Please install the current LTS.

Same with Node 14.5.1 and NPM 6.14.8

**Comment by oznu** [comment]
Logs?

**Comment by sticksen** [comment]
> Logs?

Same as in the first post. Just Sigterm without anything else.

**Comment by simonarnell** [comment]
@sticksen how minimal is your `config.json`? I was experiencing the same symptoms until I had a more complete minimal configuration file. E.g. https://github.com/homebridge/homebridge/blob/master/config-sample.json

**Comment by the-eternel** [comment]
> @sticksen how minimal is your `config.json`? I was experiencing the same symptoms until I had a more complete minimal configuration file. E.g. https://github.com/homebridge/homebridge/blob/master/config-sample.json

But you’re not using the "config" plateform. 
I have an other plugin configured in my Json and it’s still not working (I’m sure it’s the config plateform that break it)

**Comment by simonarnell** [comment]
Apologies I should have been clearer, I was referring to `config-sample.json` as a base configuration to which I added the “config” platform.

**Comment by oznu** [comment]
The logs do not show an issues with the UI. 

Ensure you have followed the setup guide: https://github.com/homebridge/homebridge/wiki/Install-Homebridge-on-Debian-or-Ubuntu-Linux

**Comment by oliverzheng** [comment]
This happened to me on Linux. I added a bunch of console logs and found that `/var/lib/homebridge` was not owned by the `homebridge` user. File permission errors by the `homebridge` package itself showed up in the logs, which I added directories for, but `homebridge-config-ui-x` wanted to touch a file at the base directory itself, and this file permissioning error did not show in the logs. Seems like there's something that's eating the error reporting.
